packages/wdio-utils/src/utils.ts

```diff
@@ -103,7 +103,7 @@ export function transformCommandLogResult (result: { file?: string, script?: str
     } else if (typeof result.script === 'string' && isBase64(result.script)) {
         return SCRIPT_PLACEHOLDER
     } else if (typeof result.script === 'string' && result.script.match(REGEX_SCRIPT_NAME)) {
-        const newScript = result.script.match(REGEX_SCRIPT_NAME)![1]
+        const newScript = result.script.match(REGEX_SCRIPT_NAME)![2]
```

Comment:
I can't tell why this changed, could you tell a little about why it changed from the second to the third value?

Reply:
Since we added an additional matcher with `(async )?` we have to now look into the 3rd element to get the function name:
![Uploading Screenshot 2023-10-11 at 1.58.35 PM.png…]()